msg/DetectedObject.msg

```diff
@@ -3,4 +3,6 @@ float32 score
 float32 left
 float32 top
 float32 right
-float32 bottom
\ No newline at end of file
+float32 bottom
+int32 img_width
+int32 img_height
```

Comment:
can we store detection in `0` to `1` range instead? so `img_width` nor `img_height` won't be required here.
e.g. with image size 640, 480, and detection on 0, 320, 0, 240, we'll store that detection as 0, 0.5, 0, 0.5 instead.

Reply:
I think we need camera width and camera height for calculation diagonal, depth, view_h_angle, and view_v_angle in the track_object function in atama

Reply:
why don't we get camera width and height from the ninshiki source of camera data instead? (is it shisen?) so basically something like this:

```
camera ---> ninshiki ----\
     \                    \
      \----------------> atama
```

Reply:
Hmm, I don't know for sure but because we just need the camera width and height, if we get a whole image data, I think it is too heavy and a little bit useless.

Reply:
i guess we should create a new topic for camera info, not just it's captured image size, but also it's depth, fov, etc.
btw, why does atama require captured width and height in the first place?